src/settings.js

```diff
@@ -62,6 +62,8 @@ const { initHathorLib } = require('./helpers/wallet.helper');
  * @property {string[]} [enabled_plugins] - The plugins we should start when
  * the service starts up.
  * @property {Record<string, PluginConfig>} [plugin_config] - The custom plugin configuration.
+ * @property {boolean} [considerHealthcheckWarnAsUnhealthy] - If true, the healthcheck
```

Comment:
issue: This new property needs to be added to the `_analizeConfig()` processing as well. See [this change on PR 356](https://github.com/HathorNetwork/hathor-wallet-headless/pull/356/files#diff-f6437b5f2e4d199e7a66f03c9d33dc5e06f5d2edc29abc401417288a43633793) for reference.

This is so that the user can update its value without the need to restart the headless application.

Reply:
I don't see why I would have to necessarily add it to `_analizeConfig()`, since this property shouldn't trigger any of the special cases treated there (stopAllWallets, reconfigLib and nonRecoverable).

It seems that a change to the property in the config would work the way it is implemented now.

If I'm missing something, I would need additional details from you.

Reply:
You're right: each call to the `healthcheck` endpoint will fetch the most updated configuration already.
Closing this thread.